frontend/taipy/package.json

```diff
@@ -30,7 +30,9 @@
     "fast-deep-equal": "^3.1.3",
     "formik": "^2.2.9",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-json-tree": "^0.19.0",
+    "taipy-gui": "file:../../taipy/gui/webapp"
```

Comment:
This line should never be committed

Reply:
> This line should never be committed

I will remove this line it probably occurred when I installed `react-json-views`